### PR TITLE
Space Travelers' Hub: an overview of what we did

### PR DESCRIPTION
**Pull request by Kingjosh007**
Here is an overview of what [Ellis](https://github.com/mwanawabangona) and [King Josaphat](https://github.com/Kingjosh007) did as partners in this React/Redux group project:

> We have to meet [the following requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md)

## Features 

As a user:

- I can see a list of all available SpaceX rockets;
- I can book each rocket by clicking a reservation button;
- I can cancel an active booking by clicking a cancel button;
- I can see a list of all current missions, along with a brief description and participation status;
- I can join a mission by clicking a button near its description;
- I can leave a mission previously joined;
- **[BONUS]** I can see a list of available dragons;
- **[BONUS]** I can click a button to reserve a dragon;
- In the "My profile" section, I can see a list of all my reserved rockets, dragons, and space missions;

## Workload distribution

 [Ellis](https://github.com/mwanawabangona) was responsible for everything related to Missions and Dragons, while [King Josaphat](https://github.com/Kingjosh007) was responsible for everything related to Rockets.

### CODE REVIEW

File: src/component/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            Mission
+          </NavLink>
+        </li>
+        <li key={4}>
+          <NavLink to="/Myprofile" className={(navData) => (navData.isActive ? 'active-link' : '')}>
+            My Profile
+          </NavLink>
+        </li>
+      </ul>
+    </nav>
```
Comment (omar-labana) [inline src/component/Navbar.js]: - [ ] I think you have misconfigured the styling, the `NavBar` is broken on the project I cloned, and the live version on `netlify`. Kindly make sure to add the proper styling and that the currently active page should be indicated with an underline.
image

File: src/component/pages/MyProfile.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const MyProfile = () => (
+  <div>
+    <h3>profile</h3>
+  </div>
+);
+export default MyProfile;
```
Comment (omar-labana) [inline src/component/pages/MyProfile.js]: - [ ] The profile page is blank. Kindly make sure to implement the required feature for the profile page:
- - [ ] Composes of two layouts and list ONLY the rockets reserved and missions joined by the user (as per design).
- - [ ] Renders a list of all joined missions that should be done filtering the state for each module.
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Kingjosh007 and @mwanawabangona, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingjosh007** [comment]
Hi @omar-labana, when we first deployed everything was fine and perfect.
We are not sure what went wrong then, but we are looking into it.

**Review by dasileker — APPROVED** [review]
## **Hi @Kingjosh007,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Approved




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**